.eslintrc.json

```diff
@@ -1,8 +1,16 @@
 {
   "plugins": [
-    "@cypress/dev"
+    "@cypress/dev",
+    "eslint-plugin"
   ],
   "extends": [
-    "plugin:@cypress/dev/general"
-  ]
+    "plugin:@cypress/dev/general",
+    "plugin:eslint-plugin/recommended"
+  ],
+  "rules": {
+    "eslint-plugin/prefer-message-ids": "off",
+    "eslint-plugin/require-meta-type": "off",
+    "eslint-plugin/require-meta-docs-description": "off",
```

Comment:
Looks like all the rules have descriptions. Why disable this?

Reply:
Copy/pasted the wrong thing, fixed.